components/Common/Banner/index.module.scss

```diff
@@ -29,24 +46,26 @@
     }
 
     &.bannerBtn {
-      background: var(--purple5);
-      border: 1px solid transparent;
-      border-radius: 5.6rem;
-      color: var(--color-fill-top-nav);
-      font-family: var(--sans);
-      font-size: 1rem;
+      border: 1.5px solid var(--color-brand-primary);
+      border-radius: var(--border-radius-3);
+      color: var(--color-brand-primary);
+      flex-shrink: 0;
+      font-size: var(--font-size-body2);
       font-style: normal;
       font-weight: var(--font-weight-semibold);
-      line-height: var(--line-height-subheading);
-      margin-right: var(--space-32);
-      padding: 0 var(--space-16);
+      height: var(--space-32);
+      margin-right: var(--space-12);
+      padding: 0 var(--space-32);
       position: relative;
       text-decoration: none;
       white-space: nowrap;
+      width: var(--space-92);
```

Comment:
This variable `--space-92` does not exist. Also no need to have a width setting here